legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/BaseGitLabApi.java

```diff
@@ -366,6 +424,11 @@ protected String getWorkspaceBranchName(String workspaceId, WorkspaceType worksp
         }
     }
 
+    protected String getPatchReleaseBranchName(VersionId patchRelaseVersionId)
+    {
+        return patchRelaseVersionId == null ? PATCH_RELEASE_BRANCH_PREFIX + BRANCH_DELIMITER : PATCH_RELEASE_BRANCH_PREFIX + BRANCH_DELIMITER + patchRelaseVersionId.toVersionIdString();
```

Comment:
If you need a method to get the prefix for patch release branches, then a separate method would be appropriate.
```suggestion
        return patchRelaseVersionId.appendVersionIdString(new StringBuilder(PATCH_RELEASE_BRANCH_PREFIX).append(BRANCH_DELIMITER)).toString();
```